custom/list.libsonnet

```diff
@@ -0,0 +1,14 @@
+{
+  core+: {
+    v1+: {
+      list: {
+        new(items):: {
```

Comment:
I thought about adding typing to this, but I couldn't find an example that allowed array and object types.

Reply:
Jsonnet does not have typing :)

However, the [docs gen](https://github.com/jsonnet-libs/docsonnet) we use allows you to specify the expected type. Please give https://github.com/jsonnet-libs/docsonnet a read and add documentation to:

- `core.v1.list`: Literal `List holds a list of objects, which may not be known by the server.`
- `core.v1.list.new`: Describe behavior
- `core.v1.list.items`: Describe behavior

If you run into any trouble, let me know and I can help you out!

Reply:
I've added the docs. PTAL.